best-practices/setting_up_rails_projects.md

```diff
@@ -0,0 +1,23 @@
+# Setting up Rails Projects
+
+Many of the DLSS software projects are [Ruby on Rails](http://rubyonrails.org/) applications. Consistency across projects can aid developers and reduce the overall maintenance burden on teams. This guide should be used as an outline on common expectations in DLSS Ruby on Rails projects. Not only should our projects reflect consistency internally, but should also reflect larger community expectations. These are not to the letter rules, but guidelines aimed to help the team.
+
+## Open source be default
+DLSS projects should be open source by default. See [licensing](/licensing.md). Developers should _never_ check in api keys or private information. Use the [config](https://github.com/railsconfig/config) gem instead. If you have any questions, check with dev-ops first.
```

Comment:
Not remembering how GitHub markdown handled relative links... do you want `/licensing.md` there or `/best-practices/licensing,md`?